File: java/spring-resilience4j/spring-resilience4j-api-a/src/main/java/com/bmarques/springresilience4japia/SpringResilience4jApiAApplication.java
package com.bmarques.springresilience4japia;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;

@SpringBootApplication
@EnableFeignClients
public class SpringResilience4jApiAApplication {

	public static void main(String[] args) {
		SpringApplication.run(SpringResilience4jApiAApplication.class, args);
	}

}

File: kotlin/spring-resilience4j/spring-resilience4j-api-a/src/main/kotlin/com/bmarques/springresilience4japia/SpringResilience4jApiAApplication.kt
package com.bmarques.springresilience4japia

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.openfeign.EnableFeignClients

@SpringBootApplication
@EnableFeignClients
class SpringResilience4jApiAApplication

fun main(args: Array<String>) {
	runApplication<SpringResilience4jApiAApplication>(*args)
}
